base/templates/base/userprofile_settings.html

```diff
@@ -0,0 +1,101 @@
+{% extends "base.html" %}
```

Comment:
I this file youre inheriting the navbar and footer from base.html while having navbar  inside the file again. 
Most of your links are not updated too.